src/Workspaces/Core/Portable/NamingStyles/Serialization/SymbolSpecification.cs

```diff
@@ -54,8 +54,6 @@ public static SymbolSpecification CreateDefaultSymbolSpecification()
                     new SymbolKindOrTypeKind(TypeKind.Interface),
                     new SymbolKindOrTypeKind(TypeKind.Delegate),
                     new SymbolKindOrTypeKind(TypeKind.Enum),
-                    new SymbolKindOrTypeKind(TypeKind.Module),
```

Comment:
Interesting. I guess that hasn't been a problem for vb yet?

Reply:
Actually, it turns out that you are correct. This change leads to "Module" check box for new VB symbol specification to be unchecked instead of checked. I will make a slight more conservative fix here.